ext/JSON3Ext.jl

```diff
@@ -0,0 +1,54 @@
+module JSON3Ext
+
+if isdefined(Base, :get_extension)
+    using JSON3
+else
+    using ..JSON3
+end
+
+using Dagger
+
+function logs_to_chrome_trace(logs::Dict)
+    execution_logs = Dict{Int,Any}()
+    Dagger.logs_event_pairs(logs) do w, start_idx, finish_idx
+        category = logs[w][:core][start_idx].category
+        if category == :compute
+            tid = logs[w][:id][start_idx].thunk_id # thunk id
+            if !haskey(execution_logs, tid)
+                execution_logs[tid] = Dict{Symbol,Any}()
+            end
+            t_start = logs[w][:core][start_idx].timestamp / 1e3 # us
+            t_stop = logs[w][:core][finish_idx].timestamp / 1e3 # us
+            proc = logs[w][:id][start_idx].processor
+            execution_logs[tid][:ts] = t_start
+            execution_logs[tid][:dur] = t_stop - t_start
+            execution_logs[tid][:pid] = proc.owner
+            execution_logs[tid][:tid] = proc.tid # thread id
+        elseif category == :add_thunk
+            tid = logs[w][:id][start_idx].thunk_id
+            if !haskey(execution_logs, tid)
+                execution_logs[tid] = Dict{Symbol,Any}()
+            end
+            f = logs[w][:timeline][start_idx].f
```

Comment:
Instead of using `:timeline` (which is preferably avoided, as it captures every single object submitted to Dagger, and can cause memory to leak), you can use `:tasknames`, which we use in GraphVizExt to get nice function names. I typically do `first(split(name, ' '))` to get just the function name, as the name is in the format `"fn [tid]"`.